src/vsCodeUtils.ts

```diff
@@ -1,8 +1,10 @@
-import { env } from "process";
-
+// @eslint-disable @typescript-eslint/no-namespace
```

Comment:
Still bunch of linting issues. I wonder if I'm the one overcomplicating this.

I feel like `@ts-ignore` inline is the best way to go. We'd only do it one time anyway. 

The other idea is we can just code `isVSCodeExtension()` to  return false and then use the define the whole expression to true in webpack.

Reply:
Refactored this after a rabbit hole + pairing for a bit. TLDR webpack plugin ordering is a mess with ts-loader, and we aren't really sure why some webpack based approaches were broken. Rather than do something hacky, we went with a simple setter that's called from the activation function.